### PR TITLE
docs: add @unpervertedkid as a contributor

### PR DESCRIPTION
**Pull request by unpervertedkid**
## Description
Practice open source contribution while taking the intro.opensauced.pizza course.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by adiati98** [comment]
Hi @unpervertedkid,
Thank you for your PR!

We have merged a PR before yours. So please [resolve the conflicts](https://github.com/open-sauced/guestbook#getting-started) first before we can merge yours :)

**Comment by unpervertedkid** [comment]
@adiati98 Done!
You can go ahead with the review

File: .all-contributorsrc
Comment (adiati98) [inline .all-contributorsrc:1008]: You added a space in these lines. Please remove the space.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @unpervertedkid,
Please fix this before we can merge your PR :)

**Comment by adiati98** [comment]
Hey @unpervertedkid 
As we're waiting for your changes, we've merged in another PR. So, please resolve the conflicts first. Thank you 🙂

**Comment by unpervertedkid** [comment]
@adiati98  Can you check it now? Sorry I took a while to respond.

**Review by adiati98 — APPROVED** [review]
Well done, @unpervertedkid! 🙌
Thank you for the changes! 😄

If you haven't, come and join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕 

git merge GIF

**Pull request merged**
